collection_requirements.rst

```diff
@@ -33,6 +33,11 @@ Be sure you're subscribed to:
 Why is this needed
 ===================
 
+Collection Infrastructure
+=========================
+
+* MUST have a publicly available issue tracker, that does not require a paid level of service to create an account or view issues.
+* MUST have a CoC (or if not fall back to Ansible's: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html )
```

Comment:
```suggestion
* MUST have a CoC (or if not fall back to Ansible's: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html )
```
i'd avoid abbreviations

Reply:
The discussion on CoC in the end went more in the direction "all collections in Ansible are governed by Ansible's CoC".